spec/authorities/qa/authorities/find_works_spec.rb

```diff
@@ -31,18 +31,18 @@
 
     context "when work has child works" do
       before do
-        work4.member_ids += [work1.id]
-        persister.save(resource: work4)
+        work1.member_ids += [work2.id]
+        persister.save(resource: work1)
       end
 
       it 'displays a list of other works deposited by current user, exluding the child work' do
-        expect(subject.map { |result| result[:id] }).to match_array [work2.id.to_s, work4.id.to_s]
+        expect(subject.map { |result| result[:id] }).to match_array [work4.id.to_s]
       end
     end
 
     context "when work has parent works" do
       before do
-        work1.member_ids += [work4.id]
+        work4.member_ids += [work1.id]
```

Comment:
I think the correct fix here is to back out this change and to change line 51 so that it is only finding `work2`. This would be consistent to how it works in master:

https://github.com/samvera/hyrax/blob/master/spec/authorities/qa/authorities/find_works_spec.rb#L48